### PR TITLE
unify the term "waiting for the next fee window"

### PR DESCRIPTION
**Pull request by CrystalBallBe**
*waiting for **the** next fee window to begin* and *waiting for next fee window to begin* are mixed.
So I unified *waiting for **the** next fee window to begin*.

### CODE REVIEW

**Comment by MicahZoltu** [comment]
I wonder if we should actually change all of these to `Waiting for Window` or something.  I think all of these are referring to a title, which really should be very pithy (short and concise).  `Waiting for the next fee window to begin` would be a proper English sentence, but I think it is too long for the title of a reporting state.

Anyone else have any thoughts on the matter?  cc @Austin-Williams

**Comment by MicahZoltu** [comment]
cc @nuevoalex who may have an opinion on the matter.

**Comment by MicahZoltu** [comment]
Since no one seems to care... would you mind changing this to `Waiting for Window` since it is being used as a title everywhere, rather than as a grammatically correct phrase?  @CrystalBallBe

**Comment by CrystalBallBe** [comment]
I like it! I thought the phrase was too long, so I like the concise and short phrase you suggested.
I will change the phrase on Friday.  (Now I'm on summer vacation.) And I appreciate your responsible support. Thanks! @MicahZoltu

**Comment by MicahZoltu** [comment]
Just comment on this PR once you have updated it with the agreed change and I'll take another look.

File: v2/english/main.tex [hunk 1]
```diff
@@ -167,15 +167,15 @@ \subsubsection{Dispute Round}
...
+In the case of a successful dispute, One of three things will happen: the market will either undergo another dispute round immediately, the market will wait until the next fee window to begin before undergoing another dispute round, or the market will enter the \textit{fork} state (Fig.~\ref{fig:reporting}e).  If the size of the filled dispute bond is greater than or equal to 2.5\% of all REP, then the market will enter the fork state. If the size of the filled dispute bond is less than 2.5\% of all REP but greater than or equal to 0.02\% of all REP, then the newly chosen outcome becomes the market's new tentative outcome and the market enters the \textit{waiting for window} phase (Fig.~\ref{fig:reporting}d). If the size of the filled dispute bond is less than 0.02\% of all REP, then the newly chosen outcome becomes the market's new tentative outcome and the market immediately enters another dispute round.
```
Comment (MicahZoltu) [inline v2/english/main.tex:170]: > the market will wait until the next fee window to begin 

should be
> the market will wait until the next fee window begins

**Comment by MicahZoltu** [comment]
One comment, but it can be fixed in a future PR and the bug existed before your PR, so I am going to merge this for now.

**Comment by CrystalBallBe** [comment]
@MicahZoltu Thanks for merging. Now I am also changing the phrase in the flowchart. I will submit a pull request later.

File: v2/english/main.tex (hunk 1 above)
Comment (CrystalBallBe) [reply v2/english/main.tex:170]: I will change that.

**Pull request merged**